verible/verilog/formatting/comment-controls.cc

```diff
@@ -136,7 +136,7 @@ void FormatWhitespaceWithDisabledByteRanges(
   if (space_text.empty() && start != 0) {
     if (!disabled_ranges.Contains(start)) {
       VLOG(3) << "output: 1*\"\\n\" (empty space text)";
-      stream << '\n';
+      stream << line_terminator_style;
```

Comment:
This looks too magic. Maybe it should just be something like `EmitLineTerminator(line_terminator_style, stream);`